### PR TITLE
Add monitoring

### PR DESCRIPTION
**Pull request by jacobwindsor**
Adds monitoring with Prometheus, Loki, and Grafana

### CODE REVIEW

**Review by MightyAx — APPROVED** [review]
Thanks for the easier review!

File: infra/helm-charts/ingest-monitoring/values.yaml [hunk 1]
```diff
@@ -0,0 +1,17 @@
+grafana:
+  persistence:
+    enabled: true
+    type: pvc
+  grafana.ini:
+    smtp:
+      enabled: true
+      host: email-smtp.us-east-1.amazonaws.com:587
+      from_address: "grafana-hca@ebi.ac.uk"
```
Comment (aaclan-ebi) [inline infra/helm-charts/ingest-monitoring/values.yaml:9]: is this mailing list existing now?

File: infra/helm-charts/ingest-monitoring/values.yaml [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+  adminUser: admin
+  adminPassword: admin
```
Comment (aaclan-ebi) [inline infra/helm-charts/ingest-monitoring/values.yaml:11]: is this a real password?

**Review by aaclan-ebi — APPROVED** [review]
cool!

File: infra/helm-charts/ingest-monitoring/values.yaml (hunk 2 above)
Comment (jacobwindsor) [reply infra/helm-charts/ingest-monitoring/values.yaml]: I fixed this

File: infra/helm-charts/ingest-monitoring/values.yaml (hunk 1 above)
Comment (jacobwindsor) [reply infra/helm-charts/ingest-monitoring/values.yaml]: Fixed and using the usual from addresss

**Review by aaclan-ebi — APPROVED** [review]

**Review by MightyAx — APPROVED** [review]

**Pull request merged**